demisto_sdk/commands/pre_commit/pre_commit_command.py

```diff
@@ -307,7 +319,7 @@ def pre_commit_manager(
         sorted((str(changed_path) for changed_path in files_to_run))
     )
 
-    logger.info(f"Running pre-commit on {files_to_run_string}")
+    logger.info(f"pre-commit received the following files: {files_to_run_string}")
```

Comment:
why change?

Reply:
@dorschw Because it's not true, here it prints the files it received in input or from git, but in practice it runs on more files

Reply:
worth a comment, so future maintainers know it too :)